client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -397,13 +424,13 @@ export class WorkspaceView extends React.Component<any, any> {
         snapshot: JSON.stringify({
           userId: Auth.userId(),
           workspaceId: workspace.id,
-          workspace: reduxBlocks.filter(b =>
+          workspace: augmentedBlocks.filter(b =>
             workspace.blocks.find(b2 => b.id === b2.id),
```

Comment:
~~This `b` is silently overwriting the `b` from line 412. Swap to `b3` to make it consistent? Same with the ones below.~~
Fine in the codebase, GitHub just confused me with their indentation.